spring-cloud-netflix-zuul/src/main/java/org/springframework/cloud/netflix/zuul/filters/SimpleRouteLocator.java

```diff
@@ -68,7 +68,13 @@ public SimpleRouteLocator(String servletPath, ZuulProperties properties) {
 		for (Entry<String, ZuulRoute> entry : getRoutesMap().entrySet()) {
 			ZuulRoute route = entry.getValue();
 			String path = route.getPath();
-			values.add(getRoute(route, path));
+			try {
+				values.add(getRoute(route, path));
```

Comment:
i guess it's better to split this lines into two 
```
ZuulRoute route  = getRoute(route, path)
values.add(route);
```
It would be more informative in case of exception.